app/src/amazonWebview/java/org/mozilla/focus/webview/FocusWebViewClient.java

```diff
@@ -11,8 +11,6 @@
 import android.net.http.SslCertificate;
```

Comment:
Ideally, we'd also clean up our existing licenses code, which I think just involves deleting the `licenses.html` file (and anything it links to that nothing else does).